Client/src/Components/EventCreation/FetchAltText.jsx

```diff
@@ -12,17 +12,19 @@ function FetchAltText({ eventId }) {
 
         return () => clearTimeout(timer);
     }, []);
+    
     //function to fetch altText from GPT 
-    let number = eventId;
+    // let number = eventId;
```

Comment:
since this line is commented out, should it be removed?

Reply:
You're right!